app/controllers/user_controller.rb

```diff
@@ -0,0 +1,5 @@
+class UserController < ApplicationController
+    def edit
+        
+    end
+end
```

Comment:
we need to have an update action, it should do something like
`current_user.update(username: params[:username])`
`redirect to '/dashboard'`